docs/upgrade.md

```diff
@@ -1,14 +1,83 @@
 # SAT Upgrade
 
-## Upgrade the SAT Product Stream
-
-Describes how to upgrade the System Admin Toolkit (SAT) product stream.
-
-### Prerequisites
-
-- CSM is installed and verified.
-- There must be at least 2 gigabytes of free space on the manager NCN on which the
-  procedure is run.
+## Install and Upgrade Framework
```

Comment:
It's a little unfortunate to duplicate the content about the IUF between install.md and upgrade.md.

Any comments I had on install.md apply here as well.